recipes/imagecodecs/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "2019.11.18" %}
+
+package:
+  name: imagecodecs
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/i/imagecodecs/imagecodecs-{{ version }}.tar.gz
+  sha256: 037e3fde4592fffdb4b8235170180d0d0ba01e4640d9660360dcc62f062e2bd1
+
+build:
+  number: 0
+  entry_points:
+    - imagecodecs=imagecodecs.__main__:main
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
Maybe?
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv --install-option="--universal""
```

Reply:
I don't think so. It needed `numpy 1.14.*` and like #10200 it needed the pin somehow.